glean-core/ios/Glean/Glean.swift

```diff
@@ -231,14 +234,20 @@ public class Glean {
 
     /// Handle foreground event and submit appropriate pings
     func handleForegroundEvent() {
-        gleanHandleClientActive()
+        if !isActive {
+            gleanHandleClientActive()
+            isActive = true
+        }
 
         GleanValidation.foregroundCount.add(1)
```

Comment:
This can then be off-by-one, right?
Guess that's fine. Maybe we should just get rid of this metric anyway.

Reply:
Yeah, it could potentially be compared to the number of baseline pings with reason=active to see how often this happens in the wild, but that would better be instrumented as a dedicated metric in an `else` branch here. As far as I know we aren't even using this for anything currently, so it could be removed.